scalableminds/webknossos
# Preserve row selection in users table

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- set up wk instance with at least two users
- open users table
- select a user
- use the search so that the unselected user is the only remaining element
- select that remaining user
- the top left should show that 2 users are selected (before the bug, only one was selected)

### Issues:
- fixes #5471

------
